simulariumio/physicell/dep/pyMCDS.py

```diff
@@ -385,15 +385,15 @@ def _read_xml(self, xml_file, parse_continuum_variables, output_path="."):
         # while we're at it, find the mesh
         coord_str = mesh_node.find("x_coordinates").text
         delimiter = mesh_node.find("x_coordinates").get("delimiter")
-        x_coords = np.array(coord_str.split(delimiter), dtype=np.float)
+        x_coords = np.array(coord_str.split(delimiter), dtype=np.float64)
```

Comment:
changes reviewed in [PR to this branch](https://github.com/simularium/simulariumio/pull/95)